pkg/services/sqlstore/migrations/peakq_rename.go

```diff
@@ -0,0 +1,30 @@
+package migrations
+
+import "github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+
+func AddPeakQRenameMigration(mg *migrator.Migrator) {
+	mg.AddMigration("Update values of apiVersion for querylibrary in resource", migrator.NewRawSQLMigration("").
+		SQLite("UPDATE resource SET \"value\"=REPLACE(\"value\", 'peakq.grafana.app/v0alpha1', 'querylibrary.grafana.app/v0alpha1') WHERE \"group\"='peakq.grafana.app';").
+		Postgres("UPDATE resource SET \"value\"=REPLACE(\"value\", 'peakq.grafana.app/v0alpha1', 'querylibrary.grafana.app/v0alpha1') WHERE \"group\"='peakq.grafana.app';").
+		Mysql("UPDATE resource SET \"value\"=REPLACE(\"value\", 'peakq.grafana.app/v0alpha1', 'querylibrary.grafana.app/v0alpha1') WHERE \"group\"='peakq.grafana.app';"))
```

Comment:
did you try without this?  I think it may work

Reply:
Yes, there is an error where somewhere on the backend it uses this value to look the types and would fail.